streams/src/test/java/org/apache/kafka/streams/integration/utils/IntegrationTestUtils.java

```diff
@@ -236,6 +238,16 @@ public static String safeUniqueTestName(final Class<?> testClass, final TestName
                 .replace('=', '_');
     }
 
+    public static String safeUniqueTestName(final Class<?> testClass, final TestInfo testInfo) {
```

Comment:
The file streams/src/test/java/org/apache/kafka/streams/integration/utils/IntegrationTestUtils.java is only added in this pull request for the build to succeed. It will be removed once https://github.com/apache/kafka/pull/12285 is merged.